install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -82,12 +82,12 @@ kubecostAggregator:
     # the federated store bucket. If set too high, there will be a delay in data
     # between the Kubecost Agents and the Aggregator.
     #
-    # Note, that the default value is set to 10m to optimize for the
+    # Note, that the default value is set to 60m to optimize for the
     # first-install experience of Kubecost (i.e. it prioritizes small data
     # becoming available more quickly).
     #
-    # default: 10m
-    DB_BUCKET_REFRESH_INTERVAL: 1h
+    # default: 60m
+    DB_BUCKET_REFRESH_INTERVAL: 2h
```

Comment:
I'm on the fence about whether to include this. With our new tuning logic for this variable in 2.3, this variable becomes even more of a foot gun than it already is

Reply:
Right, given that v2.3 will now dynamically adjust this variable, maybe we remove it from recommended configurations? I do still think we should explain what this env var is in the troubleshooting section?

Reply:
Gave it a bit more thought, and I think I'm going to leave the recommendation of `DB_BUCKET_REFRESH_INTERVAL` as is. In most cases this is the sweet spot for Enterprise-scale users. And v2.3's ability to dynamically adjust this variable should move it into a more appropriate range if needed.

If this becomes a footgun, let's remove this from the doc in future iterations!